src/pytometry/pl/_histogram.py

```diff
@@ -1,26 +1,23 @@
-from typing import Optional  # Special
-from typing import Tuple  # Classes
-
 import numpy as np
 import seaborn as sns
 from anndata import AnnData
 from matplotlib import pyplot as plt
 from matplotlib import rcParams
 
-from ..preprocessing._process_data import find_indexes
-from ..tools._normalization import normalize_arcsinh, normalize_biExp, normalize_logicle
+from pytometry.pp._process_data import find_indexes
+from pytometry.tl._normalization import normalize_arcsinh, normalize_biExp, normalize_logicle
```

Comment:
Relative imports are used everywhere except here. Is there a reason why not to use it here?
(Also, we could rename `normalize_biExp`, which does not follow the PEP naming conventions, but this is not the purpose of this PR)

Reply:
I was changing that because ruff complained about it ("absolute imports are preferable"). But no idea why it didn't complain in other places. 

> (Also, we could rename normalize_biExp, which does not follow the PEP naming conventions, but this is not the purpose of this PR)

Agree, but we should probably keep an alias for API compatibility